PocketKit/Sources/PocketKit/MyList/EmptyStates/Search/GetPremiumEmptyState.swift

```diff
@@ -0,0 +1,13 @@
+import Foundation
+import Textile
+
+struct GetPremiumEmptyState: EmptyStateViewModel {
+    let imageAsset: ImageAsset = .diamond
+    let maxWidth: CGFloat = 145
```

Comment:
not the biggest fan of hardcoded values. is there a way we can have common dimensions in their own file?

Reply:
also open to any suggestion on not hardcoding the values, but this was what I could figure out to make the UI look good. Do you mean like a constants file, something like below?
```
struct Constants {
     static let emptyStateMaxWidth: CGFloat  = 145
}
```

Reply:
yeah, but if its too much work/not something we do in our codebase, we dont have to introduce it now.

Reply:
Actually, we do I created one for our margins, so thanks for pointing this out! I updated with the commit `feedback` and will squash when it looks good :) let me know if anything else

Reply:
ugh I made a boo boo, I merged before I squashed, so I squashed this in the feature branch